src/com/Jacksonnn/DCCore/GeneralMethods.java

```diff
@@ -12,15 +12,50 @@
 
 import java.awt.*;
 import java.util.Collection;
+import java.util.HashMap;
 import java.util.Objects;
+import java.util.UUID;
 
 public class GeneralMethods {
+	private static DCCore plugin;
+
+	public GeneralMethods(DCCore plugin) {
```

Comment:
Why not use the static getter for DCCore?

Reply:
good question haha i’ll fix that tonight